client/src/App.js

```diff
@@ -11,15 +11,37 @@ import Students from "./Components/Students.js";
 import { getMessage } from "./service";
 import Mentors from "./Components/Mentors.js";
 import NewQuestion from "./Components/NewQuestion.js";
+import Results from "./Components/Results.js";
 
-import Questions from "../src/mockData/Questions.json";
+// import Questions from "../src/mockData/Questions.json";
 export function App() {
+	const [questions, setQuestions]=useState(null);
+	const [quizes, setQuizes]=useState(null);
+	if(questions!==null&&quizes!==null){
 
-	// const [message, setMessage] = useState("Loading...");
+	}
 
-	// useEffect(() => {
-	// 	getMessage().then((message) => setMessage(message));
-	// }, []);
+	const fetchAllQuestions=()=>{
+		fetch("http://localhost:3100/api/questions")
```

Comment:
You don't necessarily have to do it here, but you'll want to reference a variable instead of `http://localhost:3100` because otherwise this won't work if you're using a different port locally, or on a deployed server.

(something like `fetch(ENV['API_URL']/questions)` where `API_URL` can be set to `http://localhost:3100/api` on your machine and an IP address/domain name/whatever on production.)